@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/fn_drawIcons.sqf

```diff
@@ -44,7 +44,8 @@ private _display = uiNamespace getVariable "fza_mpd_display" get _displaySide;
 private _ctrlPoints = _display getVariable "fza_points";
 
 if (_scale == -1) then {
-    _scale = (0.125 * 5 / (_heli getVariable "fza_ah64_rangesetting"));
+    (_heli getVariable "fza_mpd_mpdState") # _displayIdx params ["","","","_state", "_persistState", ""];
```

Comment:
I will allow this as a small fix, but could you look at in a future PR removing this defaulting of scale if scale is `-1` and instead require every user to provide the scale.

That will move this into `pageTSDDraw` and allow you to get the scale more easily?